### ISSUE TITLE
Feature Request: Volume Bar

### ISSUE THREAD

**Issue by shmokinpancakes** [kmikiy/SpotMenu#234]
a volume bar added to the menu pop up for quicker volume adjustments for users that use tvs that do not support hardware volume control. (such as older tvs before 2009, HDMI1.0)

**Comment by adrienlabigne** [comment]
Neeeeeed <3
I use a lot Spotify Connect and need to change volume quickly only for Spotify, not my computer.
Thanks

**Comment by lhaeger** [comment]
Yes, please! Mostly playing Spotify on my Sonos speakers, so the macOS volume controls are useless. 

I'm currently using https://github.com/dbilgili/Ses, which controls Sonos' volume directly, but that's missing album art and the slider to navigate within a track. 

I'd love to finally see a menubar player that offers album art, volume control and in-track navigation...

**Comment by kmikiy** [comment]
closing because I'm rewriting the app from scratch https://github.com/kmikiy/SpotMenu/issues/279#issuecomment-2987565341, seams like this could be added since it looks like it's quite requested
